File: Java/2-valorPi.java
import java.util.Scanner;

public class valorAproximadoDePi {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        System.out.printf("%.5f\n", calcularPi(n));
    }

    private static double calcularPi(int n) {
        double S = 0.0;
        for (int i = 1; i <= n; i++) {
            S += 1.0 / (i * i);
        }
        return Math.pow(S * 32, 1.0 / 3);
    }
}

File: C/2-valorPi.c
#include <stdio.h>
#include <math.h>

double calculatePi(int n) {
    double S = 0.0;
    for (int i = 1; i <= n; i++) {
        S += 1.0 / (i * i);
    }
    return pow(S * 32, 1.0 / 3);
}

int main() {
    int n;
    scanf("%d", &n);
    printf("%.5lf\n", calculatePi(n));
    return 0;
}
